pkgs/by-name/me/menulibre/package.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, gnome-menus
+, gtk3
+, intltool
+, gobject-introspection
+, wrapGAppsHook
+, testers
+, menulibre
+}:
+
+python3Packages.buildPythonPackage rec {
+  name = "menulibre";
+  version = "2.2.3";
+
+  src = fetchFromGitHub {
+    owner = "bluesabre";
+    repo = "menulibre";
+    rev = "menulibre-${version}";
+    hash = "sha256-E0ukq3q4YaakOI2mDs3dh0ncZX/dqspCA+97r3JwWyA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    gnome-menus
+    psutil
+    distutils-extra
+  ];
+
+  nativeBuildInputs = [
+    gtk3
+    intltool
+    gobject-introspection
+    wrapGAppsHook
+  ];
+
+  postPatch = ''
+    sed -i "/__menulibre_data_directory__/s@(data_dir)@'$out/share/menulibre'@" setup.py
+  '';
+
+  preBuild = ''
+    export HOME=$TMPDIR
```

Comment:
Q: is there a reason for using `$TMPDIR` rather than `$(mktemp -d)` here and below in the passthru tests? grepping around in nixpkgs i see 357 uses of mktemp -d and 201 of `$TMPDIR`

Reply:
No reason in particular, I had a look at the [nix documentation](https://github.com/NixOS/nix/blob/e3febfcd532adb23ca05ac465a2b907d6f1a3529/doc/manual/src/language/derivations.md#builder-execution) and it seems like `TMPDIR` is set mostly so that builders don't accidentally interfere with each other.